src/backend/db/products.js

```diff
@@ -8,23 +15,64 @@ import { v4 as uuid } from "uuid";
 export const products = [
   {
     _id: uuid(),
```

Comment:
I suggest use any random ids instead of UUID bcz for now it's fine but when u work on single product page it will cause error bcz uuid changes on each render.